docs/en/commands/session/other/get-log-types.md

```diff
@@ -0,0 +1,107 @@
+# Get available log types
+
+Get the log for a given log type. Log buffer is reset after each request
+## Example Usage
+
+```java
+// Java
+Set<String> logEntries = driver.getLogTypes();
```

Comment:
It should be something like this ` sessionLogs.getLogTypes().contains(logType));`

Reply:
The .md files are autogenerated. This will be overwritten with:

`Set<String> logEntries = driver.manage().logs().getAvailableLogTypes();`